hotline/notifications/views.py

```diff
@@ -25,3 +25,18 @@ def create(request):
     return render(request, "notifications/create.html", {
         "form": form,
     })
+
+
+@permissions.is_active
+def list_(request):
+    if request.method == "POST":
+        form = UserSubscriptionDeleteForm(request.POST, user=request.user)
+        if form.is_valid():
+            form.save()
```

Comment:
Use `messages.success` when things work after a POST so the user knows it worked.